src/mongo/db/ldap_options.h

```diff
@@ -60,6 +60,8 @@ struct LDAPGlobalParams {
     bool ldapUseConnectionPool;
     AtomicWord<int> ldapUserCacheInvalidationInterval;
     synchronized_value<std::string> ldapQueryTemplate;
+    synchronized_value<bool> ldapDebug;
+    synchronized_value<bool> ldapReferrals;
```

Comment:
Please use `AtomicWord<bool>` for these options. This will be in line with MongoDB codebase.

Reply:
Changed.